src/Config/ObjectModel/DataSource.cs

```diff
@@ -15,6 +15,10 @@ namespace Azure.DataApiBuilder.Config.ObjectModel;
 /// <param name="Options">Custom options for the specific database. If there are no options, this could be null.</param>
 public record DataSource(DatabaseType DatabaseType, string ConnectionString, Dictionary<string, JsonElement>? Options)
 {
+
+    public const string CONN_STRING_APP_NAME_PROPERTY = "Application Name";
+    public const string CONN_STRING_APP_NAME_PROPERTY_SHORT = "App";
```

Comment:
Instead of extra constants on this record type, could you write tests that create a Sqlconnectionstringbuilder object and test the application name property directly? more self explanatory and less extra code here.

Reply:
thanks, removed redundant code.